testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -953,6 +952,8 @@ fn forked_tenure_testing(
         |config| {
             // make the duration long enough that the reorg attempt will definitely be accepted
             config.first_proposal_burn_block_timing = proposal_limit;
+            config.tenure_last_block_proposal_timeout = Duration::from_secs(0);
+
```

Comment:
How is the `forked_tenure_testing` test related to this changeset? Why did this test need to be altered, and can you describe the intended behavior changes to the test?

Reply:
I've reverted it -- it was part of me trying to un-flake this test from a while ago.